.github/workflows/ci.yml

```
@@ -37,5 +37,8 @@ jobs:
       - name: Lint
         run: yarn lint
 
+      - name: Validate example code
+        run: yarn workspace website validate:eksempler
```

```suggestion
        run: yarn validate:eksempler
```
When a script is scoped with`:`, yarn is smart enough to find it from root as long as its unique to the workspace.